modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowInstanceManagerImplTest.java

```diff
@@ -301,6 +319,29 @@ public void testUpdateActive() throws Exception {
 		}
 	}
 
+	private WorkflowInstance _completeWorkflowInstance(long classPK)
```

Comment:
Since there is only one use of this method, could you move this code to the test?